README.md

```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
 When it comes to equity compensation, everything depends on understanding some fundamental
 questions:
 
-- What is stock, and how do I own it?
-  What are shares?
-- Is stock the same as equity?
-- What is my equity worth?
-- If I have or will get stock at some point, can I sell it?
+- What is stock, and how does one own it?
+- How does stock relate to shares and equity?
+- What can you do with stock or stock options?
+- What kinds of companies utilize equity compensation?
 
-This requires discussing starting with the basics of how companies and stock work in the
-United States.
-If you think you know all this, you can [jump ahead](#equity-compensation-basics) to
-compensation.
+That last question is often overlooked or misunderstood, and it’s really where everything
```

Comment:
I still am finding this intro confusing. We don't address stock options here so don't want to mention that yet. Also I'm not sure that last question is misunderstood as much as just fundamental. How replacing the first two paragraphs with:

To understand how equity is used for compensation, it's important to understand some fundamentals about how companies are organized and owned. What are stock and shares? How is stock in a company owned? And what kinds of companies use stock for compensation?